pkg/network/domainspec/domainspec_suite_test.go

```diff
@@ -6,6 +6,7 @@ import (
 	v1 "kubevirt.io/api/core/v1"
 	api2 "kubevirt.io/client-go/api"
 	"kubevirt.io/client-go/testutils"
+
```

Comment:
Imports are supposed to be "acceptable" after running "make". I do not understand what this change does. Was it manual?

Reply:
If it was a manul whitespace add, then this is probably fixed the wrong way. "make" creates an acceptable format for thep project automatically.

Reply:
Also: since `make` produces an accepted format, there should be no reason to add such a type of linter. A linter like errcheck can make sense which checks if an error message starts with lower-case.

Reply:
https://pkg.go.dev/golang.org/x/tools/cmd/goimports

Reply:
If you want to change how imports should be sorted, just do so in https://github.com/kubevirt/kubevirt/blob/main/BUILD.bazel#L34. There is no need to run goimports again, especially not in a different configuration than we do here.

> Generally speaking, arranging the imports is a good step. It keeps things organized and the reader can easily identify internal vs external dependencies.

Generally speaking we already run goimports and format our imports to certain rules.

Reply:
I am good with using it there and dropping it from here.
Having an auto-formatter is always better.

The only issues may be the size of the change. By changing it there, it touches a large amount of files.
Here is the PR: #8050 .